packages/uniorg-parse/src/parser.ts

```diff
@@ -794,7 +809,12 @@ class Parser {
       return comment;
     }
     const value = this.r.substring(comment.contentsBegin, comment.contentsEnd);
-    return u('comment-block', { affiliated, value });
+    const [begin, end] = this.getCurrentRange(value);
```

Comment:
❗🐛 contents/value skips `#+begin_comment` and `#+end_comment`, `begin`/`end` should be wider and include them.

`getCurrentRange()` also assumes that cursor is currently at the beginning of `value`, but at this point it is after comment block.